ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main__4.18.yaml

```diff
@@ -1,12 +1,14 @@
+base_images:
+  lightspeed-service-api:
+    name: lightspeed-service-api
+    namespace: ols
+    tag: latest
```

Comment:
this is being handled differently between the 4.18 and 4.19 versions of this file, why?

(these changes to 4.18 seem to make it look like 4.16 and 4.17, so 4.19 seems like the one that's "wrong")

Reply:
@bparees Are you saying only openshift-lightspeed-service-main.yaml needs to build the service image and have this section
```
images:
- dockerfile_path: Containerfile
  to: lightspeed-service-api
```
?

Reply:
i'm saying i don't understand why it's inconsistent between them.  Either the job needs to build the image, or the image can be found in the OCP release images and thus doesn't need to be built by the job.

The answer should be the same regardless of the OCP version in question.

Reply:
There should be ONE version (currently 4.18 ) which uses the image from the PR to run the test for precommit. Rest of the versions will use the image existing image. We can either move that to 4.19 and continue to use 4.18 for precommit - and update in a new PR.